clients/memflush.cc:42:19: error: comparison between pointer and integer ('char *' and 'int')
clients/memflush.cc:51:21: error: comparison between pointer and integer ('char *' and 'int')
--- clients/memflush.cc.orig	2014-02-09 05:52:42.000000000 -0600
+++ clients/memflush.cc	2017-12-08 11:56:34.000000000 -0600
@@ -39,7 +39,7 @@
 {
   options_parse(argc, argv);
 
-  if (opt_servers == false)
+  if (opt_servers == NULL)
   {
     char *temp;
 
@@ -48,7 +48,7 @@
       opt_servers= strdup(temp);
     }
 
-    if (opt_servers == false)
+    if (opt_servers == NULL)
     {
       std::cerr << "No Servers provided" << std::endl;
       exit(EXIT_FAILURE);
